vnext/Directory.Build.props

```diff
@@ -11,6 +11,12 @@
 
     <!-- This property sets the CscTask to print the full path of the csharp file on errors and warnings rather than just the filename. -->
     <GenerateFullPaths>true</GenerateFullPaths>
+
+    <!-- 
+      SourceLink is disabled by default since customer projects are still building our projects and therefore we don't want to force customers to use this.
+      The PR (windows-vs-pr.yml) and CI (publish.yml() turn it back on.
+    -->
+    <EnableSourceLink>false</EnableSourceLink>
```

Comment:
>EnableSourceLink [](start = 5, length = 16)

Could we make it conditional if it is not set yet?

Reply:
If you specify it as a global property, msbuild will not allow it to be set ever, so when building with `/p:SourceLink=true` setting it to false will be ignored. 
I'll add the condition to be safe for customer projects, but this is generally imported early enough for any customer project to not be able to set it themselves.